scripts/run_nodes.sh

```diff
@@ -219,6 +226,9 @@ fi
 if ! command -v jq &> /dev/null; then
     error "jq could not be found on PATH!"
 fi
+if [[ "${FINALITY_VERSION}" != "current" && "${FINALITY_VERSION}" != "legacy" ]]; then
+  error "Flag finality-version should be either current or legacy."
```

Comment:
You might want to change this if you follow my earlier comment. Or not, `run_nodes.sh` might only support these two options anyway, just flagging it for consideration.

Reply:
According to my earlier answer, I don't want to.